internal/pkg/application/application.go

```diff
@@ -152,19 +153,57 @@ func (a *app) checkIfDeviceValueHasChanged(deviceId, value string) bool {
 }
 
 func getDescriptionFromDeviceState(deviceId, state string) string {
```

Comment:
Borde funktionaliteten att slå upp ett felmeddelande från en statusCode kanske egentligen ligga i iot-device-mgmt i stället? I så fall skulle felmeddelandet kunna hänga med som en property på cloudeventet som den här tjänsten tar emot?

Reply:
jag var inne på samma. Har det även i iot-agent (där det inte hör hemma, men det hängde ihop med avkodningen). Jag testar att flytta in det till iot-device-mgmt.

Reply:
fast nu har jag ändrat mig och avkodandet av felkoder ligger i iot-agent